src/main/scala/com/databricks/labs/overwatch/validation/SnapValidation.scala

```diff
@@ -60,65 +68,65 @@ class SnapValidation(params: SnapValidationParams,
    * @param sourceDatabaseName
    * @return ValidateReport
    */
-  protected def compareTable(target: PipelineTable): ValidationReport = {
-    val databaseTable = s"${params.snapDatabaseName}.${target.name}"
-    val tableName = s"${params.sourceDatabaseName}.${target.name}"
-
-    // drops columns not to be checked - some columns/structs are not always present, so we might want to avoid
-    // errors with field ordering, etc. others are random columns, so should be always dropped in all targets.
-    def dropUnecessaryCols(df: DataFrame, targetName: String) = {
-      val commonColumnsToDrop = Array("__overwatch_ctrl_noise", "Pipeline_SnapTS", "Overwatch_RunID", "__overwatch_incremental_col")
-
-      (targetName match {
-        case "spark_tasks_silver" => df.drop('TaskEndReason) // variable schema
-        case "sparkTask_gold" => df.drop('task_end_reason)  // variable schema
-        case "clusterStateFact_gold" => df.drop('driverSpecs).drop('workerSpecs) // api call or window func
-        case "jobrun_silver" |
-             "notebook_silver" |
-             "cluster_gold" |
-             "jobRun_gold" |
-             "notebook_gold" => df.drop('clusterId).drop('cluster_id) // api call or window func
-        case "job_status_silver" => df.drop('jobCluster) // api call or window func
-        case "sparkExecution_gold" => df.drop('event_log_start).drop('event_log_end) // filename can change to .gz after archival
-        case _ => df
-      }).drop(commonColumnsToDrop: _*)
-    }
-
-    try {
-      // take df1 from target (recalculated from snap), df2 from original table
-      val df1 = dropUnecessaryCols(getFilteredDF(target, tableName, startCompare, endCompare), target.name)
-      val df2 = dropUnecessaryCols(spark.table(databaseTable), target.name)
-
-      val count1 = df1.cache.count
-      val count2 = df2.cache.count
-
-      val returndf = df2.except(df1)
-        .union(df1.except(df2))
-        .dropDuplicates()
-        .select(
-          lit(tableName).alias("tableSourceName"),
-          lit(databaseTable).alias("tableSnapName"),
-          lit(count1).alias("tableSourceCount"),
-          lit(count2).alias("tableSnapCount"),
-          startCompare.alias("from"),
-          endCompare.alias("until"),
-          lit(count2-count1).alias("tableCountDiff"),
-          abs(count("*")-abs(lit(count2)-lit(count1))).alias("totalDiscrepancies"),
-          lit("processing ok").cast("string").alias("message")
-        ).as[ValidationReport]
-        .first()
-
-      df1.unpersist()
-      df2.unpersist()
-
-      returndf
-    } catch {
-      case e: Throwable =>
-        println(s"FAILED: ${databaseTable} --> ${e.getMessage}")
-        ValidationReport(tableName, databaseTable, 0L, 0L, 0L, 0L, tsTojsql(startCompare), tsTojsql(endCompare), e.getMessage)
-    }
-
-  }
+//  protected def compareTable(target: PipelineTable): ValidationReport = {
```

Comment:
commented out everything after snap step to test the pipeline and snapshot process.